src/app/components/Footer/Footer.js

```diff
@@ -1,57 +1,56 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
 import logo from '../../../assets/logo.png';
+import appConfig from '../../../config/appConfig';
+import './Footer.scss';
 
 const Footer = props => {
   const { headerFooterVisibility } = props;
   const visible = headerFooterVisibility.value;
 
+  const renderLinks = list =>
+    list.map(link => {
+      const { id, value, href, isOrigin } = link;
+      return isOrigin ? (
```

Comment:
return isOrigin ? renderRouteLink(link) : renderExternalLink(link);

I think it's more declarative